docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/advanced-usage/creating-a-whitelabel-build/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Creating a whitelabel build"
+date: "2022-09-14"
+sidebar_position: 4000
+---
+
+```mdx-code-block
+import Block5966 from "@site/docs/reusable/javascript-tracker-release-badge-v3/_index.md"
+
+<Block5966/>
+```
+
+Every time the snowplow tracker loads on the page, a queue-like object `GlobalSnowplowNamespace` is initialised on the global window scope. This object is then used to queue commands before the tracker snippet (sp.js) has fully loaded.
+
+In some cases, like two Snowplow instances on the same page from different sources or different tracker versions, there might be a need to separate this queue to avoid conflicts and tracking inconsistencies. To help you with this, we provide a custom build option for our JavaScript tracker that will produce modified loader tags (tag.js & tag.min.js) and tracker snippets (sp.js & sp.lite.js).
```

Comment:
I would clarify what these modified versions achieve. I assume they have a renamed namespace? I felt a bit uneasy reaching the end of the page and never getting my assumption confirmed :)

Reply:
Hey @stanch :) Does this seem more clear? 

To help you with this, we added a custom build option for our JavaScript tracker that will produce loader tags (tag.js & tag.min.js) and tracker snippets (sp.js & sp.lite.js) using the provided namespace and not the standard `GlobalSnowplowNamespace`.